libs/src/utils/signatures.ts

```diff
@@ -28,13 +29,13 @@ const getTransferTokensTypeHash = () => {
 // Returns the EIP712 hash which should be signed by the user
 // in order to make a call to `permit`
 function getPermitDigest (
-  web3,
-  name,
-  address,
-  chainId,
-  approve,
-  nonce,
-  deadline
+  web3: Web3,
+  name: string,
+  address: string,
+  chainId: string,
+  approve: {owner: string, spender: string, value: string},
```

Comment:
maybe this + transferTokens object could be types as well?

Reply:
yeah good idea, and would it make sense to export them so others can use them?